src/sentry/api/endpoints/project_key_details.py

```diff
@@ -47,7 +47,7 @@ def put(self, request: Request, project, key_id) -> Response:
         except ProjectKey.DoesNotExist:
             raise ResourceDoesNotExist
 
-        serializer = ProjectKeySerializer(data=request.data, partial=True)
+        serializer = ProjectKeyRequestSerializer(data=request.data, partial=True)
```

Comment:
Must be renamed b/c there is another serializer with the same name and drf-spec complains